src/components/Comments/ContainerComment.js

```diff
@@ -247,9 +247,10 @@ const ContainerComment = ({
           </View>
           <View style={{paddingLeft: 22}}>
             <View style={[styles.containerComment]}>
+              <View style={styles.lastConnectorBlocker} />
               <View style={styles.lineBeforeProfile} />
               {comments.map((item, index) => (
-                <View key={`p${index}`}>
+                <View key={`p${index}`} style={{zIndex: 10}}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The key used for the mapped elements is not unique as it depends on the index. It's better to use a unique value from each item, like an id, if available. This will prevent potential issues with React's reconciliation process when the list changes.
```diff
- <View key={`p${index}`} style={{zIndex: 10}}>
+ <View key={`p${item.id}`} style={{zIndex: 10}}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->